### ISSUE TITLE
convert MXNET to tensorflow,  npy file size is much smaller than the original MXNET model

### ISSUE THREAD

**Issue by zhangjinhong17** [microsoft/MMdnn#669]
Hi,

Platform (like ubuntu 16.04):

Python version: 3.6.8

Source framework with version ( MXNET 1.2.1 without GPU):

Destination framework with version (tensorflow 1.13.1 without GPU):

Pre-trained model path ():

Running scripts:
mmtoir -f mxnet -n R50-symbol.json -w R50-0000.params -d R50 --inputShape 3,512,512

This is the command feedback:

[21:51:25] src/nnvm/legacy_json_util.cc:209: Loading symbol saved by previous version v1.3.0. Attempting to upgrade...
[21:51:25] src/nnvm/legacy_json_util.cc:217: Symbol successfully upgraded!
/usr/local/anaconda3/envs/caffetf/lib/python3.6/site-packages/mxnet/module/base_module.py:56: UserWarning: You created Module with Module(..., label_names=['softmax_label']) but input with name 'softmax_label' is not found in symbol.list_arguments(). Did you mean one of:
	data
  warnings.warn(msg)
Warning: MXNet Parser has not supported operator null with name data.
Warning: convert the null operator with name [data] into input layer.
IR network structure is saved as [R50.json].

**and I found the R50.npy size is 560KByte, and the original MXNET model size is 118MByte**

Is anyone can help me?

Thanks

**Comment by rainLiuplus** [comment]
Hi @zhangjinhong17,
Seems that the computational graph of R50 contains a circle, which probably leads to the partial loss of the graph information during topologically sort.

**Comment by zhangjinhong17** [comment]
Hi @rainLiuplus ，
    Thanks for your help.
    Do you know how to fix it?

Thanks

**Comment by rainLiuplus** [comment]
Hi @zhangjinhong17,
Unrolling the cycle could avoid this problem (If it's possible).
